experimental/framework/tinylicious-client/src/interfaces.ts

```diff
@@ -0,0 +1,12 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export interface ITinyliciousContainerConfig {
```

Comment:
Please drop the `I` on this so it's `TinyliciousContainerConfig`. This is one of the odd things about typescript. Basically interfaces and types in typescript are interchangeable. Generally you use an interface to define an object (like this) and you use the type to augment existing types/interfaces ex `type foo = string | number;` 

This is different from most OO languages where interfaces define a class contract. 

Typescript doesn't recommend using a prefix `I` for interfaces. Our general repo practice is to use an `I` for interfaces that is expected to be implemented by a class. In this case we are using the interface to define an object someone will pass in so no `I`.

Reply:
Ah understood! Since the developer is actually passing in an object that fulfills the type, we don't need the `I` but anytime we define an interface that we expect them to actually implement the class for, the `I` is expected.

Thank you for calling out that distinction! I had been following the `I` for all interfaces rule based off of the current config interfaces in get-container. Both `TinyliciousConnectionConfig` and `TinyliciousContainerConfig` are now updated